### DISCUSSION TITLE
Import from Livraddict

### DISCUSSION THREAD

**Discussion by TheGitKoala** [bookwyrm-social/bookwyrm#2936] [Ideas]
Hello there !
I would like to join Bookwyrm, but I've been on Livraddict for years.
So, could it be possible to make import from Livraddict a thing ?
Thank you !

**Comment by mouse-reeve** [comment]
Not knowing anything about Livraddict, I'd say it's probably technically doable but realistically pretty low on the priority list to fully support that platform. That said, you may be able to import it in GoodReads mode, depending on what the output of their export looks like. Can you give an example of the headers and a couple lines of data from a Livraddict export?

**Reply by TheGitKoala** [reply]
Thanks for the reply ! I understand the low priority. I tried every export options, but none worked.
Here's how the CSV looks.

![Export Livraddict](https://github.com/bookwyrm-social/bookwyrm/assets/128918746/42126497-cfe5-488b-a9c7-1dd7d5cac6a7)

**Reply by mouse-reeve** [reply]
Ah, it's probably not working because the headers are in French. If you change those to English, it might work better. When I get home from work I can look up the right headers, please remind me if I don't reply with that by the end of the day today

**Reply by TheGitKoala** [reply]
I forgot to remember you. Could you please look when you have time ? Thank you :)

**Reply by mouse-reeve** [reply]
Okay, thank you! -- you should have `isbn`, `title`, and `author`; you'll also need to have `shelf`, which you might have to add to the file, since I don't see a column that looks analogous. You can have the value in that column be `read`, `to-read`, or `currently-reading` -- if you don't have that column with one of those values, the books won't be added to your shelves. You can also include `date started` or `date finished`. It doesn't look like you have reviews or ratings, but if you do the columns should be `review name`, `review`, and `rating` (all optional)

**Comment by TheGitKoala** [comment]
Thank you, that's so nice!
